### PR TITLE
Use window.minimize on Windows instead of window.hide

### PR DESCRIPTION
**Pull request by soutar**
Thanks @DanielPintilei for https://github.com/soutar/hyperterm-summon/pull/11.

I've updated the code a little to fix a merge conflict so I'd be grateful if you could confirm it still works as expected. Give me the thumbs up and I'll merge this. 👍

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,13 +9,17 @@ function showWindows (windows, app) {
...
-      win.hide();
+    if (win.isFullScreen()) return;
+    if (process.platform === 'win32') {
+      windows.forEach(win => win.minimize());
+    } else {
+      windows.forEach(win => win.hide());
+      app.hide(); // Mac OS only (re-focuses the last active app)
     }
   });
   app.hide(); // macOS only (re-focuses the last active app)
```
Comment (DanielPintilei) [inline index.js]: Line 25 needs to be removed.

**Comment by DanielPintilei** [comment]
Line 25 should be removed. Except that it's good to go.

**Comment by soutar** [comment]
Nice catch. I'll merge this and tag you once I've deployed 👍

**Comment by soutar** [comment]
@DanielPintilei Merged & deployed. Thanks for the PR. 😄

**Comment by DanielPintilei** [comment]
Nice! BTW don't forget to rename everything from 'Hyperterm' to 'Hyper' or 'Hyper.app' 😄

**Pull request merged**